ipa-core/src/protocol/context/dzkp_validator.rs

```diff
@@ -738,7 +738,8 @@ mod tests {
         let h3_shares: Vec<Replicated<Fp31>> = shared_inputs.iter().map(|x| x[2].clone()).collect();
 
         let futures = world
-            .malicious_contexts()
+            .contexts()
+            //.malicious_contexts()
```

Comment:
Here I think it would be better to keep the explanatory TODO comment, and remove the code.